v2.1/how-to/upgrade/from-previous-version.md

```diff
@@ -126,26 +151,50 @@ $ git clone -b $tag https://github.com/pingcap/tidb-ansible.git
     >
     > 推荐设置：`capacity` = (MEM_TOTAL * 0.5 / TiKV 实例数量)
 
+- TiKV 配置中单机多实例场景需要额外配置 `tikv_status_port` 端口：
+
+    ```
+    [tikv_servers]
+    TiKV1-1 ansible_host=172.16.10.4 deploy_dir=/data1/deploy tikv_port=20171 tikv_status_port=20181 labels="host=tikv1"
+    TiKV1-2 ansible_host=172.16.10.4 deploy_dir=/data2/deploy tikv_port=20172 tikv_status_port=20182 labels="host=tikv1"
+    TiKV2-1 ansible_host=172.16.10.5 deploy_dir=/data1/deploy tikv_port=20171 tikv_status_port=20181 labels="host=tikv2"
+    TiKV2-2 ansible_host=172.16.10.5 deploy_dir=/data2/deploy tikv_port=20172 tikv_status_port=20182 labels="host=tikv2"
+    TiKV3-1 ansible_host=172.16.10.6 deploy_dir=/data1/deploy tikv_port=20171 tikv_status_port=20181 labels="host=tikv3"
+    TiKV3-2 ansible_host=172.16.10.6 deploy_dir=/data2/deploy tikv_port=20172 tikv_status_port=20182 labels="host=tikv3"
+    ```
+
+    > **注意：**
+    >
+    > 3.0 版本单机多 TiKV 实例（进程）情况下，需要添加 `tikv_status_port` 参数。
```

Comment:
- It seems this change does not apply to `v2.1` and this 3.0 description is inappropriate for v2.1.
- Please check whether changes in https://github.com/pingcap/docs-cn/pull/1893 is updated in `docs-cn/v3.1`. If not, please help update.

Reply:
- The original`v2.1/how-to/upgrade/from-previous-version` doc (upgrading to TiDB 3.0) is deleted in [#2048](https://github.com/pingcap/docs-cn/pull/2048), so I remodify the doc accordingly.
- The changes in  #1893 are updated in [#1645](https://github.com/pingcap/docs/pull/1645/).

PTAL again. Thanks~